tests/components/sonos/test_plex_playback.py

```diff
@@ -22,99 +19,48 @@ async def test_plex_play_media(
-    requests_mock.get("https://plex.tv/users/account", text=plextv_account)
-    requests_mock.get("https://sonos.plex.tv/resources", text=plex_empty_payload)
-
-    class MockPlexServer:
-        """Mock a PlexServer instance."""
-
-        def __init__(self, has_media=False):
-            self.account = MyPlexAccount(token="token")
-            self.friendly_name = "plex"
-            if has_media:
-                self.media = "media"
-            else:
-                self.media = None
-
-        def create_playqueue(self, media, **kwargs):
-            pass
-
-        def lookup_media(self, content_type, **kwargs):
-            return self.media
-
     await setup_platform(hass, config_entry, config)
-    hass.data[PLEX_DOMAIN] = {SERVERS: {}}
     media_player = "media_player.zone_a"
-
-    # Test Plex service call with media key
-    with pytest.raises(HomeAssistantError) as excinfo:
-        assert await hass.services.async_call(
-            MP_DOMAIN,
-            SERVICE_PLAY_MEDIA,
-            {
-                ATTR_ENTITY_ID: media_player,
-                ATTR_MEDIA_CONTENT_TYPE: MEDIA_TYPE_MUSIC,
-                ATTR_MEDIA_CONTENT_ID: "plex://5",
-            },
-            True,
-        )
-    assert "No Plex servers available" in str(excinfo.value)
-
-    # Add a mocked Plex server with no media
-    hass.data[PLEX_DOMAIN][SERVERS] = {"plex": MockPlexServer()}
-
-    # Test Plex service call with dict
-    with pytest.raises(HomeAssistantError) as excinfo:
-        assert await hass.services.async_call(
-            MP_DOMAIN,
-            SERVICE_PLAY_MEDIA,
-            {
-                ATTR_ENTITY_ID: media_player,
-                ATTR_MEDIA_CONTENT_TYPE: MEDIA_TYPE_MUSIC,
-                ATTR_MEDIA_CONTENT_ID: 'plex://{"library_name": "Music", "artist_name": "Artist"}',
-            },
-            True,
-        )
-    assert "Plex media not found" in str(excinfo.value)
-
-    # Add a mocked Plex server
-    hass.data[PLEX_DOMAIN][SERVERS] = {"plex": MockPlexServer(has_media=True)}
-
-    # Test Plex service call with no Sonos speakers
-    requests_mock.get("https://sonos.plex.tv/resources", text=plex_empty_payload)
-    with pytest.raises(HomeAssistantError) as excinfo:
+    media_content_id = (
+        '{"library_name": "Music", "artist_name": "Artist", "album_name": "Album"}'
+    )
+
+    with patch(
+        "homeassistant.components.sonos.media_player.play_on_sonos"
+    ) as mock_play:
+        # Test successful Plex service call
         assert await hass.services.async_call(
             MP_DOMAIN,
             SERVICE_PLAY_MEDIA,
             {
                 ATTR_ENTITY_ID: media_player,
                 ATTR_MEDIA_CONTENT_TYPE: MEDIA_TYPE_MUSIC,
-                ATTR_MEDIA_CONTENT_ID: 'plex://{"library_name": "Music", "artist_name": "Artist"}',
+                ATTR_MEDIA_CONTENT_ID: f"{PLEX_URI_SCHEME}{media_content_id}",
             },
             True,
```

Comment:
It's nice to specify this parameter explicitly to make it easier to read what's going on.
```suggestion
            blocking=True,
```